homeassistant/components/mqtt/mixins.py

```diff
@@ -401,6 +402,7 @@ class MqttAttributes(Entity):
     """Mixin used for platforms that support JSON attributes."""
```

Comment:
We should probably make this `MqttAttributesMixin` as the linters will treat it a bit differently.  But thats for another PR

Reply:
https://github.com/home-assistant/core/pull/118039